src/crawlee/project_template/{{cookiecutter.project_name}}/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # % set extras = [cookiecutter.crawler_type]
 # % if cookiecutter.enable_apify_integration
-# % do extras.append('apify')
+apify
```

Comment:
I believe `apify` should not be there as unconditional requirement

Reply:
More precisely, the template should not be affected by this change, I believe.

Reply:
It is still in the if block. See line above and below.